<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PurchaseOrdersWindow</class>
 <widget class="QWidget" name="PurchaseOrdersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>821</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="listwidget_purchase_orders"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>120</y>
     <width>441</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_open">
        <property name="styleSheet">
         <string notr="true">min-height:1.5em;</string>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_open_po_id">
        <property name="styleSheet">
         <string notr="true">min-height:1.5em;</string>
        </property>
        <property name="text">
         <string>Open a Purchase Order with ID</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_acknowledge">
        <property name="styleSheet">
         <string notr="true">min-height:1.5em;</string>
        </property>
        <property name="text">
         <string>Acknowledge Purchase Order</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_dispatch">
        <property name="styleSheet">
         <string notr="true">min-height:1.5em;</string>
        </property>
        <property name="text">
         <string>Dispatch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_invoice">
        <property name="styleSheet">
         <string notr="true">min-height:1.5em;</string>
        </property>
        <property name="text">
         <string>Create Invoice</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>550</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btn_back">
      <property name="styleSheet">
       <string notr="true">min-height:1.5em;</string>
      </property>
      <property name="text">
       <string>Go Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>821</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_order_id">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Purchase Orders</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>821</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontallayout_pagination">
    <item>
     <widget class="QLabel" name="label_page_no">
      <property name="text">
       <string>Page 1/X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushbutton_page_forward">
      <property name="styleSheet">
       <string notr="true">min-height:1.5em;</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushbutton_page_back">
      <property name="styleSheet">
       <string notr="true">min-height:1.5em;</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
